### PR TITLE
Replace `[[ x -eq` with `(( x ==`

### PR DESCRIPTION
**Pull request by IsaacG**
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

**Comment by bkhl** [comment]
In `((` constructs you can use bare variable names, so these could be

```bash
(( status = 0 ))
```

**Comment by IsaacG** [comment]
> In `((` constructs you can use bare variable names

You certainly can. There isn't a ton of consistency or guidance on which is preferable, though. Are you saying that I ought to change them?

**Comment by bkhl** [comment]
I don't know if this has been discussed and rejected before, but I'm inclined to suggest that we stick to ShellCheck defaults, and it gives a warning if you use `$` here:

https://github.com/koalaman/shellcheck/wiki/SC2004

File: exercises/affine-cipher/affine_cipher_test.sh [hunk 1]
```diff
@@ -7,63 +7,72 @@
  @test "encode yes" {
   #[[ $BATS_RUN_SKIPPED == true  ]] || skip
   run bash affine_cipher.sh encode 5 7 "yes"
-  (( $status == 0 ))
+  (( status == 0 ))
+  (( status == 0 ))
```
Comment (bkhl) [inline exercises/affine-cipher/affine_cipher_test.sh]: Search-and-replace appears to have duplicated some lines here.

**Comment by IsaacG** [comment]
That's embarrassing. Thanks for the catch. Faulty `sed` use.

**Comment by glennj** [comment]
You broke diffie-hellman. https://travis-ci.org/github/exercism/bash/builds/665124064#L449

Can't blindly remove *all* `$`

**Comment by IsaacG** [comment]
Fixed the diffie hellman test

**Comment by glennj** [comment]
Build still failing. After you make a commit but before you push, run `bin/validate_exercises` locally.

**Comment by IsaacG** [comment]
Thanks, Glen! Got the validate script to pass locally.

**Comment by kotp** [comment]
This looks good to me, but could you squash the commits so that it is how you would like to present this merge request? I was going to do so, but realized you may have other notes now that the changes are done.

**Comment by IsaacG** [comment]
> This looks good to me, but could you squash the commits so that it is how you would like to present this merge request? I was going to do so, but realized you may have other notes now that the changes are done.

Squashed

**Comment by kotp** [comment]
Thank you @IsaacG

**Pull request merged**